modules/administration/pages/public-cloud.adoc

```diff
@@ -322,7 +322,7 @@ Then [command]``service osad stop``.
 The bootstrap should now run with a new system ID, rather than a duplicate.
 
 
-If you are onboarding Salt minion clones, then you will also need to check if they have the same Salt minion ID.
+If you are onboarding Salt client clones, then you will also need to check if they have the same Salt client ID.
```

Comment:
I think this will still be a minion ID. Might be worth checking.

Reply:
ready to push